packages/teams-js/test/private/files.spec.ts

```diff
@@ -576,15 +576,16 @@ describe('files', () => {
     it('should send the message to parent correctly', () => {
       utils.initializeWithContext('content');
 
-      const callback = jest.fn((err) => {
+      const callback = jest.fn((err, provider) => {
         expect(err).toBeFalsy();
+        expect(provider).toEqual(files.CloudStorageProvider.Dropbox);
       });
 
       files.addCloudStorageProvider(callback);
 
       const addCloudStorageProviderMessage = utils.findMessageByFunc('files.addCloudStorageProvider');
       expect(addCloudStorageProviderMessage).not.toBeNull();
-      utils.respondToMessage(addCloudStorageProviderMessage, false);
+      utils.respondToMessage(addCloudStorageProviderMessage, false, files.CloudStorageProvider.Dropbox);
```

Comment:
What does the false do here? I don't see any bool params in your function or callback

Reply:
<img width="1058" alt="image" src="https://user-images.githubusercontent.com/8016302/188928369-7d6efecc-d317-4a9e-8199-d7fb58ca6cfe.png">

As mentioned in doc comments of **toBeFalsy** method, in success scenario, we are not caring for error value and hence passed it as false instead of null for the test case.